### ISSUE TITLE
Support Publisher Ads Audits for Lighthouse

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/httparchive.org#228]
See https://developers.google.com/publisher-ads-audits

Is it possible to enable these audits in the HA tests?

cc @brendankenny @pmeenan

**Comment by pmeenan** [comment]
Doesn't seem likely.  As best as I can tell, that is a web service built on top of lighthouse and their audits are not part of lighthouse itself.  If they can publish their audits to lighthouse as optional audits then we could probably do it.

**Comment by brendankenny** [comment]
Also available as a plugin! It should be sufficient to install [`lighthouse-plugin-publisher-ads`](https://www.npmjs.com/package/lighthouse-plugin-publisher-ads) and add a `--plugins lighthouse-plugin-publisher-ads` flag to the lighthouse invocation.

It may be worth doing some spot checks for execution time and report size changes and if it's worth it for the extra audits, but as a plugin it only does audits, no additional browser gathering, so the additions should be fairly small.

cc @warrengm

**Comment by brendankenny** [comment]
> It should be sufficient to install [`lighthouse-plugin-publisher-ads`](https://www.npmjs.com/package/lighthouse-plugin-publisher-ads) and add a `--plugins lighthouse-plugin-publisher-ads` flag to the lighthouse invocation.

though from [`devtools_browser.py#L601`](https://github.com/WPO-Foundation/wptagent/blob/0b604973d4e10a964ac3e637f0ee896002e54fd9/internal/devtools_browser.py#L601) it looks like `lighthouse` might be globally installed on the test runner? If so, there is (was?) a bug that requires `lighthouse-plugin-publisher-ads` to be locally installed in that case: https://github.com/googleads/publisher-ads-lighthouse-plugin/issues/159

**Comment by pmeenan** [comment]
Yep, no particularly good reason for it to have to be installed globally but that's how they're all currently configured.  I'm watching the issue and can re-visit once that is fixed (or take a look at installing lighthouse locally instead - but that is a bigger deployment change).

**Comment by romaincurutchet** [comment]
I would like to see the results from this audit in a table. Working with hundreds of publishers, it is particularly useful for me to see how much of their Cumulative Layout Shift issues is due to ads, which I can't get elsewhere.

**Comment by max-ostapenko** [comment]
@pmeenan is this something feasible already?

**Comment by pmeenan** [comment]
Doesn't look like there are any technical restrictions. We'd mostly have to decide if we're opening up the lighthouse tests to run with plugins and how we decide what plugins we choose to include (and our comfort level with ongoing support since the plugin hasn't been touched in a year - maybe it just doesn't need updates).

**Comment by max-ostapenko** [comment]
Now that I looked into it - there are indeed maintenance issues.
It doesn't support lighthouse versions after 10.0.1 (and we obviously want the latest).
As for me, it's too much effort to consider updates+support in such case.

Very unfortunate, especially since there are even particular audits checking for ads impact on LCP.

@jburger424 Is there a chance of getting a new release?
